src/components/photoPlayer/plugin.js

```diff
@@ -1,23 +1,18 @@
-define(['browser', 'require', 'events', 'apphost', 'loading', 'dom', 'playbackManager', 'appRouter', 'appSettings', 'connectionManager'], function (browser, require, events, appHost, loading, dom, playbackManager, appRouter, appSettings, connectionManager) {
-    'use strict';
-
-    function PhotoPlayer() {
-
-        var self = this;
-
-        self.name = 'Photo Player';
-        self.type = 'mediaplayer';
-        self.id = 'photoplayer';
-
-        // Let any players created by plugins take priority
-        self.priority = 1;
+import connectionManager from 'connectionManager';
+
+export class PhotoPlayer {
+    constructor() {
+        this.name = 'Photo Player';
+        this.type = 'mediaplayer';
+        this.id = 'photoplayer';
+        this.priority = 1;
     }
 
-    PhotoPlayer.prototype.play = function (options) {
+    play(options) {
 
         return new Promise(function (resolve, reject) {
 
-            require(['slideshow'], function (slideshow) {
+            import('slideshow').then(slideshow => {
```

Comment:
_Same as in #1089_
`TypeError: slideshow is not a constructor`
`slideshow = {default: ƒ}`

```suggestion
            import('slideshow').then(({default: slideshow}) => {
```

Reply:
Or here is an alternative suggestion https://github.com/jellyfin/jellyfin-web/pull/1089#discussion_r432112537
_if possible_